### PR TITLE
add more unit tests for sysadmin vs end user

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
This PR adds more unit tests for the actions menu

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-43345

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/actions_menu/actions_menu.test.tsx [hunk 1]
```diff
@@ -67,24 +67,47 @@ describe('components/actions_menu/ActionsMenu', () => {
...
         const wrapper = shallowWithIntl(
             <ActionsMenu {...baseProps}/>,
         );
         wrapper.setProps({
             isSysAdmin: false,
         });
-        expect(wrapper.find('#marketPlaceButton').exists()).toBe(false);
+
+        // menu should be empty
+        expect(wrapper.debug()).toMatchSnapshot();
```
Comment (larkox) [inline components/actions_menu/actions_menu.test.tsx:107]: Just for my knowledge: why debug here?

**Review by larkox — APPROVED** [review]

**Review by Willyfrog — APPROVED** [review]

File: components/actions_menu/actions_menu.test.tsx (hunk 1 above)
Comment (jfrerich) [reply components/actions_menu/actions_menu.test.tsx:107]: `.debug()` will print the contents of the rendered component, which is empty in this case.

**Comment by Willyfrog** [comment]
@jfrerich what are the next steps on this PR?

**Comment by jfrerich** [comment]
@Willyfrog, only need to merge now.  Thanks.

**Pull request merged**